site/content/entry/citra-progress-report-2020-q2.md

```diff
@@ -0,0 +1,699 @@
+And in the worst case scenario, the game doesn't use that object for seconds or even minutes, until when it finally does and out of nowhere, crash!
+
+Since this is so new, you can bet that there will be bugs and broken loads, so we ask for your help in reporting these issues when you find them.
+
+(Gif of save state loading in pokemon)
+That's not video looping, it's just me reloading a save state whenever my attack misses!
+
+## New 3DS Support
+
+You may have known this already, but New 3DS support has finally arrived!
+
+New 3DS support is another highly requested feature for Citra. The New 3DS is a member of the Nintendo 3DS family with upgraded hardware. It has more cores and more RAM, so games can do more computing. There are a handful of New 3DS exclusives, and many more that are enhanced.
+
+With the combined efforts of [B3n30](https://github.com/B3n30), [FearlessTobi](https://github.com/FearlessTobi), and many other developers, we were finally able to deliver proper New 3DS support. We saw almost all major New 3DS exclusive games booting and running in a playable state. Although not all of them run at fullspeed yet, we hope you will be able to enjoy exploring the huge worlds of Xenoblade Chronicles 3D or fight some zombies in the (very pixelated) 3DS port of Minecraft.
+
+{{< figure src="/images/entry/citra-progress-report-2020-q2/new_citra_xl.jpg"
+    title="Liked the New Citra XL? The functionality was real!" >}}
+
+### Core timing 2.0 ([#4913](https://github.com/citra-emu/citra/pulls/4913)) by [B3n30](https://github.com/B3n30)
+
+The biggest roadblock for New 3DS support was our scheduler. Since most Old 3DS games only used one of the two cores present, we emulated one core, and left the other one basically untouched.
+
+This worked for most games, but broke some, even in Old 3DS mode. A long time ago, we tried to mitigate this and other threading bugs with a "priority boost" feature that would run threads that haven't been run in a long time for a bit. As our scheduling got better though, this hack was eventually removed and some games were just left broken.
+
+On the New 3DS, things got even worse. There are 4 cores to be emulated, and our only emulating one of them proved to be even bigger of a problem. Many games required multiple cores to run correctly, and they just refused to boot at all in older versions of Citra.
+
+It required quite a bit of work to get the cores in sync while not ruining performance. Developer [B3n30](https://github.com/B3n30) researched the scheduler and stepped up to implement what we call "Core timing 2.0". He added a timer for each core, and basically rewrote how our timing worked, thus creating a more accurate scheduler. After this change, not only was a big step done towards compatibility with New 3DS games, but it also fixed all the games that were broken with the priority boost hack removed, for instance, [Game].
```

Comment:
For the first wording change, I don't have a strong opinion. For the second change, you can use inverted sentences for `not only ... but also ...`

Reply:
> 
> 
> For the first wording change, I don't have a strong opinion. For the second change, you can use inverted sentences for `not only ... but also ...`

In my opinion the second change with the word "was" was only made by me because it sounded gramatically incorrect to me.